sdk/src/main/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/sdk/model/response/TransactionInfoOrderStatus.java

```diff
@@ -0,0 +1,61 @@
+package nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.response;
+
+import kong.unirest.json.JSONObject;
+
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.Money;
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.enums.TransactionStatus;
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.enums.TransactionType;
+
+import java.util.Optional;
+
+/**
+ * this class contains details about a transaction related to order status.
+ */
+public class TransactionInfoOrderStatus {
```

Comment:
Please rename this into TransactionInfoStatus, no need for mentioning "Order" here. What I noticed there is only a minor difference between TransactionInfo and TransactionInfoStatus. Will see to discus it with @calindumitru and @kkreuning so that we only reuse existing one instead of having two of them.